src/main/kotlin/com/terraformation/backend/accelerator/api/DeliverablesController.kt

```diff
@@ -0,0 +1,207 @@
+            id = deliverableId,
+            internalComment = "These guys are real jokers.",
+            name = "Incorporation Documents",
+            organizationId = OrganizationId(1),
+            organizationName = "Test Org",
+            participantId = ParticipantId(2),
+            participantName = "Random Participant",
+            projectId = ProjectId(3),
+            projectName = "Omega Project",
+            status = SubmissionStatus.Rejected,
+            type = DeliverableType.Document,
+        ))
+  }
+
+  @ApiResponse(
+      responseCode = "307",
+      description =
+          "If the current user has permission to view the document, redirects to the document " +
+              "on the document store. Depending on the document store, the redirect URL may or " +
+              "may not be valid for only a limited time.")
+  @ApiResponse404
+  @GetMapping("/{deliverableId}/documents/{documentId}")
+  @Operation(summary = "Gets a single submission document from a deliverable.")
+  fun getDeliverableDocument(
+      @PathVariable deliverableId: DeliverableId,
+      @PathVariable documentId: SubmissionDocumentId
+  ): RedirectView {
+    val url = URI("https://dropbox.com/")
+
+    val redirectView = RedirectView(url.toString(), false, false)
+    redirectView.setStatusCode(HttpStatus.TEMPORARY_REDIRECT)
+
+    return redirectView
+  }
+}
+
+data class ListDeliverablesElement(
+    val category: DeliverableCategory,
+    @Schema(description = "Optional description of the deliverable in HTML form.")
+    val descriptionHtml: String?,
+    val id: DeliverableId,
+    val name: String,
+    @Schema(
+        description =
+            "Number of documents submitted for this deliverable. Only valid for deliverables of " +
+                "type Document.")
+    val numDocuments: Int?,
+    val organizationId: OrganizationId,
+    val organizationName: String,
+    val participantId: ParticipantId,
+    val participantName: String,
+    val projectId: ProjectId,
+    val projectName: String,
+    val status: SubmissionStatus,
+    val type: DeliverableType,
+)
+
+data class SubmissionDocumentPayload(
+    val createdTime: Instant,
+    val description: String,
+    val documentStore: DocumentStore,
+    val id: SubmissionDocumentId,
+    val name: String,
+)
+
+data class DeliverablePayload(
+    val category: DeliverableCategory,
+    @Schema(description = "Optional description of the deliverable in HTML form.")
+    val descriptionHtml: String?,
+    val documents: List<SubmissionDocumentPayload>,
+    @Schema(
+        description =
+            "If the deliverable has been reviewed, the user-visible feedback from the review.")
+    val feedback: String?,
+    val id: DeliverableId,
+    @Schema(
+        description =
+            "Internal-only comment on the submission. Only present if the current user has accelerator admin privileges.")
+    val internalComment: String?,
```

Comment:
Another approach would be for the admin view to use a separate API endpoint to fetch deliverables, but I thought keeping the API as small as possible was the better tradeoff. Please give feedback if that's the wrong call!

Reply:
This works for the client, implementation so far has been assuming a single API endpoint to list deliverables.

Reply:
I just added a question in the PRD about this field, the UI seems to indicate that this is _multiple_ comments. I think this is good as-is for now, and hopefully for MVP we can just allow multiple comments within the individual field (which it looks like the underlying column will support).